### PR TITLE
Merge release 17.7 beta 4 into develop

### PR DESCRIPTION
**Pull request by mokagio**
Includes:

- Jetpack app review link fix #16811 by @momo-ozawa 
- Jetpack fix when tapping on notification #16812 by @jleandroperez 

Nothing to test, if CI Shows green, we're good.

## Regression Notes
1. Potential unintended areas of impact
N.A.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N.A.

3. What automated tests I added (or what prevented me from doing so)
N.A.

---

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes. **N.A.**
- [x] I have considered adding accessibility improvements for my changes. **N.A.**
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary. **N.A.**

### CODE REVIEW

**Review by mokagio — COMMENTED** [review]
- [x] Version update in `.xcconfig`
- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress
- [x] Diffs from the PRs that made it into this beta

File: config/Version.public.xcconfig [hunk 1]
```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=17.7
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=17.7.0.2
+VERSION_LONG=17.7.0.3
```
Comment (AliSoftware) [inline config/Version.public.xcconfig:4]: I always get caught for a second when your PRs says "beta 4" and I see ".0.3" here until I remember that iOS version numbers are zero-based 😅 👍

**Review by AliSoftware — APPROVED** [review]
:shipit:

File: config/Version.public.xcconfig (hunk 1 above)
Comment (mokagio) [reply config/Version.public.xcconfig:4]: Me too!

I'll try and say "beta 0" in 17.8 and see what people say 🙊

**Pull request merged**